charts/rpe-service-auth-provider/values.yaml

```diff
@@ -60,4 +60,5 @@ java:
         - microservicekey-rd-professional-api
         - microservicekey-rd-user-profile-api
         - microservicekey-ethos-repl-service
+        - microservicekey-employment-tribunals
```

Comment:
you probably want to bump the chart version as well

Reply:
done